apps/microtvm/gemmini/template_project/src/add.c

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
src files for each project type should be in a separate sub-directory. Please follow the same convention.

Reply:
Thanks, I applied the suggestion.